static/bidder-params/adsyield.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Adsyield Adapter Params",
+  "description": "A schema which validates params accepted by the Adsyield adapter",
+  "type": "object",
+
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "Ad network's RTB host",
+      "format": "hostname",
+      "pattern": "^.+\\..+$"
+    },
```

Comment:
Why do you need the host name?

Reply:
We need it for internal routing because we can no longer use subdomains. And we need to support backward compatibility

Reply:
You can read more about it here: https://github.com/prebid/prebid-server/pull/2539#discussion_r1095070010
We changed this behaviour here: https://github.com/prebid/prebid-server/pull/2653/files#diff-267b5e4d799f4ba6dea77765c24de75d3a77e1c40f5936f9a0033260790ccb87R1